packages/client/src/checkout/draftOrders/types/draftOrder.types.ts

```diff
@@ -0,0 +1,67 @@
+import type {
+  CheckoutOrderDeliveryBundle,
+  DraftOrderItem,
+} from '../../index.js';
+
+import type { Metadata } from '../../../types/index.js';
+
+export enum DraftOrderStatus {
+  Created = 'Created',
+  Canceled = 'Canceled',
+  Completed = 'Completed',
+}
+
+type DraftOrderAddress = {
```

Comment:
This type should be exported as it is used in a type that is exported as well. Also, can this type be derived from one of the other already existing address types? Is this address different from an address of a checkout order? If not, then I suggest reusing that type here.

Reply:
Exported.
And yes, this address is different from a checkout order address.
I tried to reuse the checkout order address type, but it needs some manipulation. To avoid complexity, I decided to create a new one only for draftOrders.

I don’t know if this is suppose to be different, but I will check with the BE team